ktor-server/ktor-server-core/jvm/test/io/ktor/tests/features/CORSTest.kt

```diff
@@ -40,4 +48,43 @@ class CORSTest {
         every { point.port } returns port
         return point
     }
+
+    @Test
+    fun expectRequestHeadersCheckPassIfControlRequestHeadersHeaderHasEmptyValue(): Unit = runBlocking {
```

Comment:
please make sure that the test starts with word `test`, consider also making short name.

I would use: `testCorsIsEmpty`